components/resources/brave_shields_strings.grdp

```diff
@@ -18,7 +18,7 @@
 
   <message name="IDS_BRAVE_SHIELDS_BROKEN" desc="A footnote below the main Shields switch button.">
     If this site seems broken, try Shields down. 
-    <ph name="LINK_BEFORE">$1</ph>Note: this may reduce Brave privacy protections.<ph name="LINK_AFTER">$2</ph>
+    <ph name="LINK_BEFORE">$1</ph>Note: this may reduce Brave's privacy protections.<ph name="LINK_AFTER">$2</ph>
```

Comment:
This isn't really a link is it? Just styling differently for the second line? Also I'm not sure how translations will handle or preserve the line-break (and whether we should rely on that anyway).
I suggest we split in to 2 strings to be simpler.